tests/codegen/fcode/scripts/imports.py

```diff
@@ -1,8 +1,9 @@
-from expressions import ai
-xi = ai + 1
-
-from expressions import ad, bd
-xd = ad * bd
+#from expressions import ai
+#xi = ai + 1
+#
+#from expressions import ad, bd
+#xd = ad * bd
+xi = 2
```

Comment:
Why commenting those lines out?

Reply:
I did not put the declarations of the variables into the module. Handling variables requires an extra step where the main is called as an `init` function. The variables would also need to be saved as members of the module so that the module takes care of the declaration. This is possible but would require more work. If we want to be able to import variables then I suggest we leave this for a second PR.

Reply:
This is not an urgent matter, we can discuss about it later.